### DISCUSSION TITLE
Client certificate

### DISCUSSION THREAD

**Discussion by 8uk54nt0** [turtle0x1/LxdMosaic#438] [Q&A]
I installed lxdmosaic via snap.
Now i want to connect to my LXD host but mosaic keeps saying "LXD client certificate is not trusted...".
I have to trust mosaic's client certificate on my LXD via "lxc config trust add ..." but i can't get the mosaic's client certificate?
Where can i find it?
Thank you.

**Comment by turtle0x1** [comment]
LXDMosaic generates the certificates on the fly for the host, it would take some "magic" todo it the other way round (esp in SNAP).
